corehq/apps/translations/validator.py

```diff
@@ -157,7 +157,7 @@ def _compare_sheet(self, module_or_form_id, uploaded_rows, for_type):
         }
         if None in header_indices.values():
             raise BulkAppTranslationsException(_("Could not find column(s) '{}'").format(
-                ", ".join([str(k) for k, v in header_indices.items() if v == None])
+                ", ".join([str(k) for k, v in header_indices.items() if v is None])
```

Comment:
is v == None incorrect otherwise?

Reply:
It works, it's just bad style, possibly because you can't use similar operators like `<=` and `>=` with None in python 3: https://docs.python.org/3/whatsnew/3.0.html#ordering-comparisons